neurons/miners/miner.py

```diff
@@ -102,11 +102,15 @@ def __init__(self, config=None):
             forward_fn=self.query,
             blacklist_fn=self.query_blacklist,
             priority_fn=self.query_priority,
+        ).attach(
+            forward_fn=self.cypher_query,
```

Comment:
the cypher query is the query endpoint, you should implement the `query()` function